cloudflare/resource_cloudflare_access_policy.go

```diff
@@ -86,6 +86,13 @@ var policyOptionElement = &schema.Resource{
 				Type: schema.TypeString,
 			},
 		},
+		"service_token": {
```

Comment:
I'm torn on this as to whether or not to pluralise it because it accepts multiple values. Without pluralising it and iterating over the values, you'll need to provide multiple of these blocks to achieve the same thing.

```hcl
resource "cloudflare_access_policy" "test_policy" {
  application_id = "cb029e245cfdd66dc8d2e570d5dd3322"
  zone_id        = "d41d8cd98f00b204e9800998ecf8427e"
  name           = "staging policy"
  precedence     = "1"
  decision       = "allow"

  include {
    service_token = ["abcd1234"]
    service_token = ["4321dcba"]
  }
}
```

vs

```hcl
resource "cloudflare_access_policy" "test_policy" {
  application_id = "cb029e245cfdd66dc8d2e570d5dd3322"
  zone_id        = "d41d8cd98f00b204e9800998ecf8427e"
  name           = "staging policy"
  precedence     = "1"
  decision       = "allow"

  include {
    service_tokens = [
      "abcd1234",
      "4321dcba"
    ]
  }
}
```

Reply:
The naming convention is just following the syntax as in other params like email / email_domain. For me is clearer to not putting an s as a param.

Furthermore, it works with both 
```
  include {
    service_token = ["abcd1234"]
    service_token = ["4321dcba"]
  }
```

or

```
  include {
    service_token = [
      "abcd1234",
      "4321dcba"
    ]
  }
```

Reply:
If it can be multiple, it makes more sense that we pluralise the directive to convey the intention.

Reply:
So is that implies to change other params (email, email_domain, ip) to be plural as well? 
I am afraid it could introduce backward incapabilities.

Reply:
🤔 that raises a good point; I didn’t realise you were referring to this resource. 👍 for following what is present for the others. we can always rename later if needed.